src/services/codefixes/helpers.ts

```diff
@@ -195,6 +197,20 @@ namespace ts.codefix {
             if (method) addClassElement(method);
         }
 
+        function createModifiers(): NodeArray<Modifier> | undefined {
+            let modifiers: Modifier[] | undefined;
+
+            if (modifierFlags) {
+                modifiers = combine(modifiers, factory.createModifiersFromModifierFlags(modifierFlags));
+            }
+
+            if (overridesAbstract) {
+                modifiers = append(modifiers, factory.createToken(SyntaxKind.OverrideKeyword));
+            }

```

Comment:
Per https://github.com/microsoft/TypeScript/issues/50464#issuecomment-1264730161, this change would make `override` only be added if `noImplicitOverride` is enabled.

```suggestion
            if (overridesAbstract && context.program.getCompilerOptions().noImplicitOverride) {
                modifiers = append(modifiers, factory.createToken(SyntaxKind.OverrideKeyword));
            }
```

I don't personally use the flag or abstract classes in any of my active projects, and am not sure what the intent of `override` is long-term. Is `override` intended only for projects that opt-in, or as a general good best practice?

Reply:
I'd say `noImplicitOverride` is a pretty good signal of whether someone wants to consistently see an `override` or not in their programs. So I think we should only add the keyword if the flag is set, otherwise we risk this being too annoying.  @RyanCavanaugh what do you think?